src/schema/v2/me/myCollectionDeleteArtworkImageMutation.ts

```diff
@@ -0,0 +1,59 @@
+import { GraphQLString, GraphQLNonNull } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import { MyCollectionArtworkMutationType } from "./myCollection"
+import { formatGravityError } from "lib/gravityErrorHandler"
+
+export const MyCollectionDeleteArtworkImageMutation = mutationWithClientMutationId<
```

Comment:
This might be better called as the more generic `deleteArtworkImageMutation`? What do you think? It's more of an implementation detail that the Gravity endpoint will only let a user delete the images of an artwork they're authorized for.